src/Components/Activities/Activities.js

```diff
@@ -23,7 +28,7 @@ const Activities = ({ activities, updateActivity }) => {
     <section className="activity-page">
       <div className="activity-card">
         {currentActivityIndex > 0 && <button onClick={() => prevActivity()}>←</button>}
-        <h2>{activities[currentActivityIndex].activity}</h2>
+        <h2>{activities[currentActivityIndex]?.activity}</h2>
```

Comment:
ooh! I haven't seen this syntax before, is this checking for truthiness? Nice error handling catch.  

update: I pulled it down to play around with it, and I'm noticing that it throws an error without this "?". Even on refresh when the state is set to zero, (so the currentActivityIndex is set to zero) which to me would say it should be trying to access the activity property of the first index of the activities array, which is "Go find a spider" which does exist. What am I missing here?

Reply:
I have the same question here - would love for you to share in standup this morning!